cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/hdfs/HdfsStorage.java

```diff
@@ -0,0 +1,39 @@
+package com.linkedin.openhouse.cluster.storage.hdfs;
+
+import com.linkedin.openhouse.cluster.storage.BaseStorage;
+import com.linkedin.openhouse.cluster.storage.StorageClient;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * The HdfsStorage class is an implementation of the Storage interface for HDFS storage. It uses a
+ * HdfsStorageClient to interact with the HDFS file system. The HdfsStorageClient uses an Apache
+ * Hadoop FileSystem to interact with the HDFS file system.
```

Comment:
The HdfsStorageClient uses an Apache Hadoop FileSystem to interact with the HDFS file system.
Not following this comment.

Reply:
got it! replaced the confusing string with class link.